### PR TITLE
initial storage.conf handling

### PR DESCRIPTION
**Pull request by maxamillion**
Signed-off-by: Adam Miller <admiller@redhat.com>

### CODE REVIEW

**Comment by richm** [comment]
[citest commit:73d27d782bdfbd7c1e4b5ee63ce7fda611377d76]

**Review by richm — APPROVED** [review]
lgtm

**Comment by richm** [comment]
> lgtm

spoke too soon - needs a README.md entry for the parameter - looks like I missed that for the containers.conf and registries.conf PRs too . . .

File: defaults/main.yml [hunk 1]
```diff
@@ -59,3 +59,9 @@ podman_containers_conf: {}
 # __podman_registries_conf_system or __podman_registries_conf_user
 # paths, respectively, depending on execution mode selected
 podman_registries_conf: {}
+
+# containers-storage.conf(5) settings, these will be provided in a drop-in
```
Comment (vrothberg) [inline defaults/main.yml:63]:
```suggestion
# containers-storage.conf(5) settings, these will be provided in a drop-in
```

Probably a copy-paste:  storage.conf does not provide drop-in support.

Comment (maxamillion) [reply defaults/main.yml:63]: Ah ok, then that's going to get complicated. I'll have to rethink this and follow the pattern that the ssh system role did.

**Comment by richm** [comment]
> > 
> 
> should I just fix all three in a separate docs PR?

Either that, or just fix all three in this PR.  Whatever you want to do.

**Comment by richm** [comment]
@maxamillion any updates?

**Comment by richm** [comment]
closing in favor of https://github.com/linux-system-roles/podman/pull/19

**Pull request closed without merging**